book/run/sync-op-mainnet.md

```diff
@@ -1,5 +1,25 @@
 # Sync OP Mainnet
 
+To sync OP mainnet, bedrock state needs to be imported as a starting point. There are currently two ways:
+
+* Minimal bootstrap: only state is imported without any OVM historical data.
+* Full bootstrap: state, blocks and receipts are imported.
+
+## Minimal bootstrap
+
+**Bedrock state is required.** It can be exported from [op-geth](https://github.com/testinprod-io/op-erigon/blob/pcw109550/bedrock-db-migration/bedrock-migration.md#export-state) (**.jsonl**) or downloaded directly from [here](https://mega.nz/file/GdZ1xbAT#a9cBv3AqzsTGXYgX7nZc_3fl--tcBmOAIwIA5ND6kwc).
```

Comment:
can we rephrase this that it becomes obvious that this is just the state snapshot at the bedrock block